### DISCUSSION TITLE
Changing size of graphs and option to display filled line graphs

### DISCUSSION THREAD

**Discussion by emp-00** [raiguard/ModernGadgets#280] [Q&A] [answered]
Q1) I would like to increase the size of "line graphs" in the ModernGadgets suite. See example, screenhot CPU meter: How to e.g. double the size of (only) the graph? Text & numbers should stay as they are.
![image](https://user-images.githubusercontent.com/60714349/153719223-b284ab14-965b-4a32-8b20-cc16abf933a7.png)

Q2) Can line graphs be drawn "filled" instead of just a thin line: How to configure this option? See example from a gadget "8GadgetPack" - I would like to see CPU Meter draw the trend like this for better visibility of CPU load problems.
![image](https://user-images.githubusercontent.com/60714349/153719557-1c5b6ed6-a3dd-434c-8fbf-98cfc41cd6e1.png)

Many thanks for your help and the great suite of ModernGadgets!

**Comment by raiguard** [comment, marked as answer]
1: You can add a variable `graphHeight=whatever` (default is 42) to the global settings file. You can access this file using the "edit settings file" button in the global settings.
![image](https://user-images.githubusercontent.com/3515394/153781398-1aa3bc02-014b-4ce6-842a-d0ea6a24bf56.png)

2: That is an entirely separate meter type called `Histogram`, and would be a bit more involved. Disks Meter uses them for the read/write time.

**Reply by emp-00** [reply]
Thank you @raiguard - 
1) Works like a charm, excactly what I was looking for. 
2) Found the option "Read/Write Time Histograms" but this does not have any effect on my graph. I see only the line graphs of my five disks - no "histogram" visible. Have refreshed the skin as well, no change.

**Reply by raiguard** [reply]
The histograms will show read/write time, not disk activity. Start performing a huge file transfer and you'll probably see it show up.

**Reply by emp-00** [reply]
Indeed - with large transfers a "shadow" appears, see screenshot (I copied from H: to G:). It's not a shadow under the line graph but rather the full graph height is filled. Is this intended or a bug? I was looking for a graph with filled color _only in the _area below the line__.
![image](https://user-images.githubusercontent.com/60714349/154126971-4693ef12-6b9b-4b77-b8ee-76937ff657c5.png)

**Reply by raiguard** [reply]
It's intended, because it's showing an entirely different measurement (read/write time percentage) instead of the usage.

What I intended by pointing that out is that you'll have to edit the skin to convert those to be disk usage instead of disk read/write time. What you're looking for is not included in the gadgets by default so you'll have to hack it in.

**Reply by emp-00** [reply]
Thank you - understood. Will need to take some time, I already "hacked" an RSS reader to suit my needs - took many hours... I fear this one will also take hours. Rainmeter code is not really intuitive, honestly. But still much appreciate your work, those Modern Gadgets suite is fantastic!!
